src/python/WMComponent/JobUpdater/JobUpdaterPoller.py

```diff
@@ -42,7 +42,7 @@ def __init__(self, config):
         self.config = config
 
         self.bossAir = BossAirAPI(config = self.config)
-        self.reqmgr = RequestManager({'endpoint' : self.config.JobUpdater.reqMgrUrl})
+        self.reqmgr2 = ReqMgr({'endpoint' : self.config.JobUpdater.reqMgr2Url})
```

Comment:
I'm failing to instantiate this class. It seems you need to provide a simple string instead of a dict (or change __init__ in ReqMgr).